doc/user-guide/indexing.rst

```diff
@@ -536,6 +533,31 @@ __ https://docs.scipy.org/doc/numpy/user/basics.indexing.html#assigning-values-t
       da.isel(x=[0, 1, 2])[1] = -1
       da
 
+You can also assign values in all variables of a ``Dataset`` at once:
+
+.. ipython:: python
+
+    ds = xr.tutorial.open_dataset("eraint_uvz")
+    ds
+
+    # by integer
+    ds[dict(longitude=10, latitude=10)]["u"]
+    ds[dict(longitude=10, latitude=10)] = 1
+    ds[dict(longitude=10, latitude=10)]["u"]
+
+    # by label
+    ds.loc[dict(longitude=47.25, level=200, latitude=[11.25, 12])]["u"]
+    ds.loc[dict(longitude=47.25, level=200, latitude=[11.25, 12])] = 100
+    ds.loc[dict(longitude=47.25, level=200, latitude=[11.25, 12])]["u"]
+
+    # dataset as new values
+    new_dat = ds.loc[dict(longitude=48, level=200, latitude=[11.25, 12])]["u"]
```

Comment:
that's not a `Dataset`, is it? I think `["u"]` returns a `DataArray`?

Reply:
yes right. I actually wanted to take the full dataset not only u. I changed it.